docker/Dockerfile

```diff
@@ -18,12 +18,12 @@ RUN echo "## Start building" \
         python3-pip \
     && echo "## Done"
 
-RUN echo "## Get EMSDK" \
-    && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
-    && echo "## Done"
+# Copy the contents of this repository to the container
+COPY . ${EMSDK}
```

Comment:
I my version of this change I used `ADD . /emsdk`: https://github.com/emscripten-core/emsdk/compare/docker_use_local_emsdk?expand=1#diff-ebacf6f6ae4ee68078bb16454b23247dR26

I guess they are the same .. i can't remember why I chose one over the other.

Reply:
It doesn't make much difference: https://stackoverflow.com/a/24958548/10952119. I think `COPY` is fine here.